product_docs/docs/pgd/5/cli/command_ref/pgd_switchover.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: switchover
+---
+
+Switchover to new write leader.
+
+### Synopsis
+
+Switchover to new write leader. Use switchover method fast for immediate
```

Comment:
Same comment as above: Switches over? Should "fast" be marked with back ticks (i.e., code)?

Reply:
yes both fixed used single quote though